packages/react/config/esmInput.js

```diff
@@ -0,0 +1,50 @@
+  'ssr/index': 'src/ssr/index.ts',
+  'components/Accordion/index': 'src/components/accordion/index.ts',
+  'components/Button/index': 'src/components/button/index.ts',
+  'components/Card/index': 'src/components/card/index.ts',
+  'components/Checkbox/index': 'src/components/checkbox/index.ts',
+  'components/Columns/index': 'src/components/columns/index.ts',
+  'components/CookieConsent/index': 'src/components/cookieConsent/index.ts',
+  'components/Combobox/index': 'src/components/dropdown/combobox/index.ts',
+  'components/Container/index': 'src/components/container/index.ts',
+  'components/DateInput/index': 'src/components/dateInput/index.ts',
+  'components/Dialog/index': 'src/components/dialog/index.ts',
+  'components/ErrorSummary/index': 'src/components/errorSummary/index.ts',
+  'components/Fieldset/index': 'src/components/fieldset/index.ts',
+  'components/FileInput/index': 'src/components/fileInput/index.ts',
+  'components/Footer/index': 'src/components/footer/index.ts',
+  'components/ImageWithCard/index': 'src/components/imageWithCard/index.ts',
+  'components/Koros/index': 'src/components/koros/index.ts',
+  'components/Link/index': 'src/components/link/index.ts',
+  'components/Linkbbox/index': 'src/components/linkbox/index.ts',
+  'components/LoadingSpinner/index': 'src/components/loadingSpinner/index.ts',
+  'components/Logo/index': 'src/components/logo/index.ts',
+  'components/Navigation/index': 'src/components/navigation/index.ts',
+  'components/Notification/index': 'src/components/notification/index.ts',
+  'components/NumberInput/index': 'src/components/numberInput/index.ts',
+  'components/Pagination/index': 'src/components/pagination/index.ts',
+  'components/PasswordInput/index': 'src/components/passwordInput/index.ts',
+  'components/PhoneInput/index': 'src/components/phoneInput/index.ts',
+  'components/RadioButton/index': 'src/components/radioButton/index.ts',
+  'components/SearchInput/index': 'src/components/searchInput/index.ts',
+  'components/Section/index': 'src/components/section/index.ts',
+  'components/Select/index': 'src/components/dropdown/select/index.ts',
+  'components/SelectionGroup/index': 'src/components/selectionGroup/index.ts',
+  'components/SideNavigation/index': 'src/components/sideNavigation/index.ts',
+  'components/StatusLabel/index': 'src/components/statusLabel/index.ts',
+  'components/Stepper/index': 'src/components/stepper/index.ts',
+  'components/Table/index': 'src/components/table/index.ts',
+  'components/Tabs/index': 'src/components/tabs/index.ts',
+  'components/Tag/index': 'src/components/tag/index.ts',
+  'components/TextInput/index': 'src/components/textInput/index.ts',
+  'components/Textarea/index': 'src/components/textarea/index.ts',
+  'components/TimeInput/index': 'src/components/timeInput/index.ts',
+  'components/ToggleButton/index': 'src/components/toggleButton/index.ts',
+  'components/Tooltip/index': 'src/components/tooltip/index.ts',
```

Comment:
When the scaffold script has been run, the component is added to the very last line. However all these are in alphabetical order. This makes me wonder that is there a script (maybe in rollup or somewhere) that puts these in order during build or something? And now that these are in this new file, are they ordered anymore? Or should we put the new line in correct place alphabetically. It's not exactly a deal breaker but it is nice 😄

Reply:
Sorting on scaffold is relatively easy to do: https://github.com/City-of-Helsinki/helsinki-design-system/pull/1011/commits/cd1eea70a0acb65562912cded4c8d43b46be30e8